package.json

```diff
@@ -34,6 +35,7 @@
     "babel-preset-es2015": "6.9.0",
     "babel-preset-react": "6.11.1",
     "chai": "3.5.0",
+    "jest": "20.0.4",
```

Comment:
Since jest isn't used by the project's tests, just the examples, could you move this devDependency to examples/package.json?